tests/plotting/test_plotting.py

```diff
@@ -3811,3 +3811,98 @@ def test_voxelize_volume():
     # Create a voxel volume from unequal density dimensions and plot result.
     vox = pv.voxelize_volume(mesh, density=[0.15, 0.15, 0.5])
     vox.plot(scalars='InsideMesh', show_edges=True, cpos=cpos)
+
+
+def test_paraview_2d_style():
+    def setup_plot():
+        mesh = pv.Cube()
+        mesh["face_id"] = np.arange(6)
+        pl = pv.Plotter()
+        pl.enable_paraview_2d_style()
+        pl.enable_parallel_projection()
+        pl.add_mesh(mesh, scalars="face_id")
+        return pl
+
+    # baseline, image
+    pl = setup_plot()
+    pl.show()
+
+    start = (100, 100)
+    pan = rotate = (150, 150)
+    spin = (100, 150)
+    dolly = (100, 50)
+
+    # left click pans, image 1
```

Comment:
Checking if the image is in 2D view in the test is a great idea. Please explain in your comments what images you expect to be displayed by each operation. I can generally describe what we check in the images, but comments would give me more confidence.

Reply:
I added some comments, hopefully it makes sense.  I also exaggerated more the dolly images since those were more subtle.